app/models/concerns/pre_sale_prices/line_item_pre_saleable.rb

```diff
@@ -0,0 +1,20 @@
+module PreSalePrices
+  module LineItemPreSaleable
+    extend ActiveSupport::Concern
+
+    included do
+      before_validation :copy_pre_sale_price
```

Comment:
In that case use `if pre_sale_price.nil?` when assigning value like in the original implementation. This will stop it being updated.